examples/002_beambeam/008_beambeam6d_collective_beamstrahlung.py

```diff
@@ -0,0 +1,363 @@
+beta_x_sext_right = 3
+beta_y_sext_right = 5000
+
+alpha_x_sext_left  = 0
+alpha_y_sext_left  = 0
+alpha_x_sext_right = 0
+alpha_y_sext_right = 0
+
+#crab sextupoles
+k2_left  = k2_factor / (2 * phi * beta_y * beta_y_sext_left ) * np.sqrt(beta_x / beta_x_sext_left )
+k2_right = k2_factor / (2 * phi * beta_y * beta_y_sext_right) * np.sqrt(beta_x / beta_x_sext_right)
+
+el_sextupole_left  = xt.Multipole(order=2, knl=[0, 0, k2_left], length=0.3)
+el_sextupole_right = xt.Multipole(order=2, knl=[0, 0, -k2_right], length=0.3)
+
+# from IP to right crab sextupole (sy2r.2)
+el_arc_left_b1 = xt.LinearTransferMatrix(_context=context,
+    Q_x = 0,  # 2pi phase advance so integer part is zero
+    Q_y = 0.25,  # 2.5pi
+    Q_s = 0,  # no dipole here so no synchrotron motion
+    beta_x_0 = beta_x,
+    beta_y_0 = beta_y,
+    beta_x_1 = beta_x_sext_left,
+    beta_y_1 = beta_y_sext_left,
+    alpha_x_0 = 0,
+    alpha_y_0 = 0,
+    alpha_x_1 = alpha_x_sext_left,
+    alpha_y_1 = alpha_y_sext_left,
+    beta_s = beta_s,
+)
+
+# between 2 sextupoles
+el_arc_mid_b1 = xt.LinearTransferMatrix(_context=context,
+    Q_x =  qx,
+    Q_y =  qy - 0.5, # subtract .25*2 phase advance from small arcs
+    Q_s = -qs,
+    beta_x_0 = beta_x_sext_left,
+    beta_y_0 = beta_y_sext_left,
+    beta_x_1 = beta_x_sext_right,
+    beta_y_1 = beta_y_sext_right,
+    alpha_x_0 = alpha_x_sext_left,
+    alpha_y_0 = alpha_y_sext_left,
+    alpha_x_1 = alpha_x_sext_right,
+    alpha_y_1 = alpha_y_sext_right,
+    beta_s = sigma_z/sigma_delta,
+    damping_rate_x = damping_rate_x,
+    damping_rate_y = damping_rate_y,
+    damping_rate_s = damping_rate_s,
+    equ_emit_x = physemit_x,
+    equ_emit_y = physemit_y,
+    equ_emit_s = physemit_s, # only here i need sigma_z delta SR
+    energy_increment = u_bs*1e9, # U_BS for one IP
+)
+
+# from left crab sextupole to IP2 (sy2l.1)
+el_arc_right_b1 = xt.LinearTransferMatrix(_context=context,
+    Q_x = 0,  # 2pi phase advance so integer part is zero
+    Q_y = 0.25,  # 2.5pi
+    Q_s = 0,  # no dipole here so no synchrotron motion
+    beta_x_0 = beta_x_sext_right,
+    beta_y_0 = beta_y_sext_right,
+    beta_x_1 = beta_x,
+    beta_y_1 = beta_y,
+    alpha_x_0 = alpha_x_sext_right,
+    alpha_y_0 = alpha_y_sext_right,
+    alpha_x_1 = 0,
+    alpha_y_1 = 0,
+    beta_s = beta_s,
+)
+
+# injection from initial distribution to right sextupole
+el_inject_b1 = xt.LinearTransferMatrix(_context=context,
+    beta_x_0 = beta_x,
+    beta_y_0 = beta_y,
+    beta_x_1 = beta_x_sext_right,
+    beta_y_1 = beta_y_sext_right,
+    alpha_x_0 = 0,
+    alpha_y_0 = 0,
+    alpha_x_1 = alpha_x_sext_right,
+    alpha_y_1 = alpha_y_sext_right,
+    beta_s = beta_s,
+)
+
+##########################################
+# weakstrong beambeam with beamstrahlung #
+##########################################
+
+bin_edges = sigma_z_tot*np.linspace(-3.0,3.0,n_slices+1)
+slicer = xf.Slicer(n_slices=n_slices, sigma_z=sigma_z_tot, mode="improved")
```

Comment:
"improved" does not mean much, can we use a more expressive name?

Reply:
renamed to shatilov

https://mafurman.lbl.gov/mafurman.lbl.gov/LBL-37680.pdf